api4/ldap_test.go

```diff
@@ -5,12 +5,16 @@ package api4
 
 import (
 	"testing"
+
+	"github.com/mattermost/mattermost-server/model"
 )
 
 func TestTestLdap(t *testing.T) {
 	th := Setup().InitBasic()
 	defer th.TearDown()
 
+	th.App.SetLicense(model.NewTestLicense("ldap_groups"))
```

Comment:
Could we add the test case for `th.SystemAdminClient.TestLdap()` failing without the license?